test/parallel/test-assert.js

```diff
@@ -630,19 +567,13 @@ common.expectsError(
   {
     code: 'ERR_ASSERTION',
     type: assert.AssertionError,
-    generatedMessage: true,
-    message: 'The expression evaluated to a falsy value:\n\n  ' +
-             'assert.ok(null)\n'
   }
 );
 common.expectsError(
   () => assert(typeof 123 === 'string'),
   {
     code: 'ERR_ASSERTION',
     type: assert.AssertionError,
-    generatedMessage: true,
-    message: 'The expression evaluated to a falsy value:\n\n  ' +
```

Comment:
Instead of always showing `false != true` or something like that, it would be good to actually keep the `The expression evaluated to a falsy value` part.